src/OSPSuite.Core/Domain/Services/ObservedDataTask.cs

```diff
@@ -36,6 +36,7 @@ public interface IObservedDataTask
       /// </summary>
       bool DeleteAll();
 
+      void UpdateMolWeight(DataRepository observedData);
```

Comment:
this would deserve a space between the definition (resharper to the rescue) but even more so  a summary explaining what this is going to do. Can you add this?

Reply:
formated with resharper and summary added